src/dbnode/persist/fs/files.go

```diff
@@ -468,7 +468,11 @@ func forEachInfoFile(
 			digestsFilePath = filesetPathFromTimeAndIndex(dir, t, volume, digestFileSuffix)
 			infoFilePath = filesetPathFromTimeAndIndex(dir, t, volume, infoFileSuffix)
 		}
-		if !FileExists(checkpointFilePath) {
+		checkpointExists, err := FileExists(checkpointFilePath)
+		if err != nil {
+			continue
```

Comment:
This kind of sucks, but thats how we handle errors in this function right now. Should address in a separate P.R

Reply:
#854

Reply:
Sounds good.